core-java/src/main/java/eu/ostrzyciel/jelly/core/JellyConstants.java

```diff
@@ -0,0 +1,31 @@
+package eu.ostrzyciel.jelly.core;
+
+public class JellyConstants {
+
+    private JellyConstants() {}
+
+    public static final String JELLY_NAME = "Jelly";
+    public static final String JELLY_FILE_EXTENSION = "jelly";
+    public static final String JELLY_CONTENT_TYPE = "application/x-jelly-rdf";
+
+    /**
+     * @deprecated Use {@link #PROTO_VERSION_1_0_X} instead.
```

Comment:
Remove all deprecated stuff.

Reply:
For context: the first release with jelly-jvm rewritten in Java will be 3.0.0

Reply:
doned